client/packages/core/src/Reactor.js

```diff
@@ -518,23 +519,30 @@ export default class Reactor {
           }
           // Handles add-triple|retract-triple
           // If in mapping, we update the attr-id
-          const [action, eid, attrId, ...rest] = txStep;
-          const newTxStep = mapping[attrId]
-            ? [action, eid, mapping[attrId], ...rest]
-            : txStep;
+          const newTxStep = instaml.rewriteStep(mapping, txStep);
+
           retTxSteps.push(newTxStep);
           return [mapping, retTxSteps];
         },
         [mapping, []],
       );
+      if (chunks) {
+        try {
+          return [newMapping, instaml.transform(attrs, chunks)];
```

Comment:
Found a problem with doing transform to rewrite the mutations--it will recreate the`add-attr` every time you call it, so you won't keep consistent attr ids for the new attrs. I think I'll go back to the existing method and find a different way to solve the links issue.

Reply:
Resolved. Now we're just augmenting the existing rewriter.